src/subdomains/supporting/log/log-job.service.ts

```diff
@@ -75,7 +60,72 @@ export class LogJobService {
     if (DisabledProcess(Process.TRADING_LOG)) return;
 
     // trading log
-    const tradingLog = await this.tradingRuleService.getCurrentTradingOrders().then((t) =>
+    const tradingLog = await this.getTradingLog();
+
+    // assets
+    const assets = await this.assetService.getAllAssets().then((l) => l.filter((a) => a.type !== AssetType.CUSTOM));
+
+    // asset log
+    const assetLog = await this.getAssetLog(assets);
+
+    // balances sorted by financialType
+    const balancesByFinancialType = this.getBalancesByFinancialType(assets, assetLog);
+
+    const plusBalanceChf = Util.sumObjValue(Object.values(balancesByFinancialType), 'plusBalanceChf');
+    const minusBalanceChf = Util.sumObjValue(Object.values(balancesByFinancialType), 'minusBalanceChf');
+
+    await this.logService.create({
+      system: 'LogService',
+      subsystem: 'FinancialDataLog',
+      severity: LogSeverity.INFO,
+      message: JSON.stringify({
+        assets: assetLog,
+        tradings: tradingLog,
+        balancesByFinancialType,
+        balancesTotal: {
+          plusBalanceChf,
+          minusBalanceChf,
+          totalBalanceChf: plusBalanceChf - minusBalanceChf,
+        },
+      }),
+    });
+  }
+
+  // --- LOG METHODS --- //
+
+  private getBalancesByFinancialType(assets: Asset[], assetLog: any): BalancesByFinancialType {
+    const financialTypeMap = Util.groupBy<Asset, string>(
```

Comment:
Maybe we should define a type/interface for the asset log?

Reply:
Haha ups, yeah I already defined it, but forgot to change it here 😁